ocaml/xapi/xapi_globs.ml

```diff
@@ -1023,6 +1023,8 @@ let python3_path = ref "/usr/bin/python3"
 let observer_experimental_components =
   ref (StringSet.singleton Constants.observer_component_smapi)
 
+let pool_recommendations_dir = "/etc/xapi.pool-recommendations.d"
```

Comment:
Please follow the pattern used for the other values in this file, which are `ref`s with a default value, which can be overridden by xapi.conf. For this case, you can append it to `nonessential_dirs` at the end of the file.